testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/RepoScriptBlockUtil.groovy

```diff
@@ -184,7 +189,7 @@ class RepoScriptBlockUtil {
 
                 @CompileDynamic
                 void applyToAllProjects(Gradle gradle, Closure projectClosure) {
-                    if (gradle.gradleVersion >= "8.8") {
+                    if (GradleVersion.version(gradle.gradleVersion) >= GradleVersion.version("8.8")) {
```

Comment:
@blindpirate Can you fix the comparison to 4.4 below as well? I think I made the mistake in copying that instead of switching on my brain when doing this comparison.

Reply:
@blindpirate I moved this change and added also fixed the other use of `gradle.gradleVersion` in:
* https://github.com/gradle/gradle/pull/29518

Reply:
please merge that and rebase this branch afterwards